uiowa/uiowa
# HOTFIX - fixed existing blocks with grid settings

Resolves https://github.com/uiowa/uiowa/issues/4949

# How to Test

- Sync a bunch of sites to test an update hook that runs on the following blocks:
  - 'inline_block:uiowa_articles'
  - 'inline_block:uiowa_events'
  - 'inline_block:uiowa_people'
  - 'inline_block:uiowa_aggregator'
  - 'views_block:article_list_block-list_article'
  - 'views_block:people_list_block-list_card'
- There should be no update errors.
- Additionally you should be able to go into an existing block that is displaying as a list, change it to grid and see that the 3-column option is set. Switch back to list and save the block. The markup should not include the 3-column class.
- On a new block with grid display option, see that the 3-column option is set and that there is not an empty option.
- You should be able to save and re-save blocks and get the expected display settings regardless of what you choose.
